SocketRocket/SRWebSocket.m

```diff
@@ -570,6 +570,33 @@ - (void)_initializeStreams;
     
     _inputStream.delegate = self;
     _outputStream.delegate = self;
+    
+    NSString *networkServiceType;
+    switch (_urlRequest.networkServiceType) {
+        case NSURLNetworkServiceTypeDefault:
+            return;
+        case NSURLNetworkServiceTypeVoIP: {
+#ifdef __IPHONE_9_0
+            // voice over IP control - this service type is deprecated in favor of using PushKit for VoIP control
+            if (floor(NSFoundationVersionNumber) > NSFoundationVersionNumber_iOS_8_3) {
```

Comment:
This seems like it would create unexpected behavior if one is on a higher version and expects it to be a VoIP service (even if they ignore deprecation warnings).

What if we just logged a warning inside a dispatch_once saying its deprecated, but we're going to do it anyways?